content/applications/finance/fiscal_localizations/india.rst

```diff
@@ -46,6 +46,13 @@ localization:
 
 .. _india/e-invoicing:
 
+Indian Configuration
+====================
+
+In :menuselection:`Settings --> Users & Companies --> Companies`, add your :guilabel:`PAN` and
+:guilabel:`GSTIN`. The :guilabel:`PAN` is essential for determining the type of taxpayer,
+while :guilabel:`GSTIN`is required for generating e-Invoices and E-waybills.
```

Comment:
```suggestion
:guilabel:`GSTIN`. The PAN is essential for determining the type of taxpayer,
while GSTIN is required for generating e-Invoices and E-waybills.
```
Guilabels aren't necessary in this case, and the second guilabel caused a _ci/documentation_ error due to a missing space after the second backtick :)

Reply:
Hello @xpl-odoo 
made the suggested changes.